live-site/components/mentors-section/MentorsSection.styles.tsx

```diff
@@ -62,10 +69,96 @@ const StyledMentorsFilterRadio = styled.div`
   }
 `;
 
+const StyledMentorsListContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  padding-bottom: 20px;
+  justify-content: center;
+  max-width: 95%;
+  margin: auto;
+
+  @media ${max.tabletLg} {
+    max-width: 500px;
```

Comment:
nitpick but could we change everything to use em's to be standard with the rest of our repo?